<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Image 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Image 2</string>
    </property>
   </widget>
   <widget class="ImageView" name="imageView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="ImageView" name="imageView_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="ImageView" name="imageView_1_edit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="ImageView" name="imageView_2_edit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>311</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Image Components</string>
    </property>
    <item>
     <property name="text">
      <string>Choose FT Components</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FT Magnitude</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FT Phase</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FT Real Component</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FT Imaginary Component</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Image Components</string>
    </property>
    <item>
     <property name="text">
      <string>Choose FT Components</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FT Magnitude</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FT Phase</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FT Real Component</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FT Imaginary Component</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Window"/>
   </widget>
   <widget class="QMenu" name="menuBrowse">
    <property name="title">
     <string>Browse</string>
    </property>
    <addaction name="actionImage1"/>
    <addaction name="actionImage2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBrowse"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionImage1">
   <property name="text">
    <string>Image1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionImage2">
   <property name="text">
    <string>Image2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
